templates/gitlab-multi-runner/1/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '2'
+
+services:
+
+  gitlab-runner-config:
+    image: gitlab/gitlab-runner:alpine-v9.4.1
+    stdin_open: true
+    volumes:
+    - /etc/gitlab-runner/
+    tty: true
+    command:
+    - register
+    - -n
+    - --url
+    - ${GITLAB_URL}
+    - --registration-token
+    - ${GITLAB_TOKEN}
+    - --tag-list
+    - ${GITLAB_TAGS}
+    - --executor
+    - docker
+    - --description
+    - Rancher Docker Runner
+    - --docker-image
+    - docker:latest
+    - --docker-volumes
+    - /var/run/docker.sock:/var/run/docker.sock
+    - --docker-privileged
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
```

Comment:
If scheduler is global, please, use optional host_label to control where to run the service, if you like to, using go tempalting. 
```
+      io.rancher.scheduler.global: 'true'
+    {{- if ne .Values.host_label ""}}
+      io.rancher.scheduler.affinity:host_label: ${host_label}
+    {{- end}}
```